docs/source/installed_packages.rst

```diff
@@ -70,4 +70,7 @@ mypy and used for type checking.
 
 If you use mypy to type check a Python other than the version running mypy, you
 can use the ``--python-executable`` flag to point to the executable, and mypy
-will find packages installed for that python executable.
\ No newline at end of file
+will find packages installed for that python executable.
+
+Note that mypy does not support some more dynamic import features, such as zip
+imports and advanced custom import hooks.
```

Comment:
Zip imports are not dynamic (though they are advanced). I'd s/dynamic/advanced/ and s/advanced// on the next line.